VERSIONING.md

```diff
@@ -0,0 +1,35 @@
+# Versioning
+This site uses [Docusaurus' built in support for versioning](https://docusaurus.io/docs/versioning). This is configured so that:
+
+* */docs/mainnet/* represents the last deployed version on [mainnet](https://blog.vega.xyz/what-to-expect-from-restricted-mainnet-616086d9fdaf)
+* */docs/testnet/* represents the most recent version on [Fairground testnet](https://fairground.wtf)
+
+Both of those do not neccessarily move in lock step. There could be two version updates to one network before the other updates. Bumping a testnet version is easy. Moving a testnet version to mainnet involves the steps below:
```

Comment:
comment (non-blocking): It may be easy but it would be worth describing how. But that can be done in another iteration.

Reply:
#152